src/flutter/shell/platform/linux_embedded/window/native_window_drm_eglstream.cc

```diff
@@ -0,0 +1,304 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux_embedded/window/native_window_drm_eglstream.h"
+
+#include <sys/mman.h>
+#include <unistd.h>
+
+#include <cstring>
+
+#include "flutter/shell/platform/linux_embedded/logger.h"
+#include "flutter/shell/platform/linux_embedded/surface/cursor_data.h"
+
+namespace flutter {
+
+NativeWindowDrmEglstream::NativeWindowDrmEglstream(const char* deviceFilename)
+    : NativeWindowDrm(deviceFilename) {
+  if (!valid_) {
+    return;
+  }
+
+  if (!ConfigureDisplayAdditional()) {
+    valid_ = false;
+    return;
+  }
+
+  // DRM-NVDC doesn't have drmIsMaster() function.
+  // Master permissions are granted by default to all clients.
```

Comment:
Really? What about the situation where weston has already worked?

Reply:
From the following document, drmIsMaster() function is not provided.
https://docs.nvidia.com/drive/nvvib_docs/NVIDIA%20DRIVE%20Linux%20SDK%20Development%20Guide/baggage/group__direct__rendering__manager.html#ga3f9326af9fc8eddc23dc6e263a2160a1

Also, with weston running, flutter-embedded-linux with backend = DRM-EGLStream could be started. (It seems that weston has finished and flutter-embedded-linux has started instead)